packages/gitbook/src/components/TableOfContents/TableOfContents.tsx

```diff
@@ -35,8 +35,10 @@ export function TableOfContents(props: {
     ancestors: Array<RevisionPageDocument | RevisionPageGroup>;
     header?: React.ReactNode;
     headerOffset: { sectionsHeader: boolean; topHeader: boolean };
+    innerHeader?: React.ReactNode;
```

Comment:
If we can put a comment on what's the difference between `header` and `innerHeader` :)